lib/simulator-xcode-6.js

```diff
@@ -1189,6 +1189,60 @@ class SimulatorXcode6 extends EventEmitter {
 
   //endregion
 
+  /**
+   * @typedef {Object} ProcessInfo
+   * @property {number} pid The actual process identifier.
+   * Could be zero if the process is the system one.
+   * @property {?string} group The process group identifier.
+   * This could be `null` if the process is not a part of the
+   * particular group. For `normal` application processes the group
+   * name usually equals to `UIKitApplication`.
+   * @property {string} name The process name, for example
+   * `com.apple.Preferences`
+   */
+
+  /**
+   * Lists processes that are currently running on the given Simulator.
+   * The simulator must be in running state in order for this
+   * method to work properly.
+   *
+   * @return {Array<ProcessInfo>} The list of retrieved process
+   * information
+   * @throws {Error} if no process information could be retrieved.
+   */
+  async ps () {
+    const {stdout} = await this.simctl.spawnProcess([
+      'launchctl',
+      'print',
+      'system',
+    ]);
+
+    const servicesMatch = /^\s*services\s*=\s*{([^}]+)/m.exec(stdout);
```

Comment:
` xcrun simctl spawn booted launchctl list`
> list            Lists information about services.

I compared the result and `services = {...}` in the `print system`, then their count was the same. (Not checked one by one)

Reply:
could be. I would say both options are fine